MockGUI/GolemModel.cs

```diff
@@ -79,14 +79,14 @@ public static async Task<GolemViewModel> Load(string modulesDir, RelayType relay
         public static async Task<GolemViewModel> CreateStatic(string modulesDir, RelayType relayType)
         {
             var loggerFactory = LoggerFactory.Create(builder =>
-                // builder.AddFile("golem.log")
-                builder.AddConsole()
+                builder.AddFile(Path.Combine(modulesDir, "golem.log"))
```

Comment:
Put this at least in golem data dir, not modules

Reply:
Moved.